src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -974,6 +974,11 @@ def __init__(self, config):
         self.gpt_neox = GPTNeoXModel(config)
         self.qa_outputs = nn.Linear(config.hidden_size, 2)
 
+        # Model parallel
+        self.model_parallel = False
+        self.device_map = None
+        self.gradient_checkpointing = False
```

Comment:
```suggestion
```
This part should be deleted, this model does not implement the old model parallel